wurst/event/DamageType.wurst

```diff
@@ -8,16 +8,11 @@ import DamageDetection
 import OnUnitEnterLeave
```

Comment:
Please undo all the whitespace changes in this file... I've opened a different PR to fix the tabs/spaces thing, but it just bloats this PR.

Reply:
My PR has been merged into master now.